electora_py/api.py

```diff
@@ -1,11 +1,33 @@
-from nucypher.characters.lawful import Bob, Ursula
+from typing import Optional, Dict
+from gql import gql, Client
+from gql.transport.aiohttp import AIOHTTPTransport
+from nucypher_core import ferveo
+
+from nucypher.characters.lawful import Ursula, Bob
 from nucypher.cli.utils import connect_to_blockchain
 from nucypher.utilities.emitters import StdoutEmitter
-from nucypher_core import ferveo
+
+# Networks
 
 _GOERLI_URI = "https://goerli.infura.io/v3/663d60ae0f504f168b362c2bda60f81c"
 _TEACHER_URI = "https://lynx.nucypher.network:9151"
 
+# GraphQL
+
+_ELECTORA_ARWEAVE_TAG = "electora/ballot/uuid"
+_ARWEAVE_GQL_ENDPOINT = "https://arweave.net/graphql"  # TODO: Get this from somewhere else (config, env, round-robin, etc.)
```

Comment:
Can we please not use comments for this? If something needs doing, create an issue for it.

Reply:
I suppose it's a principled debate, but IMO this indictes the exact entry point that a change needs to occur (in fact I'd like to update this comment to include the issue number).  Anyways, I'll remove it.  Opened #2.